docs/package-hub/package-builder/gem-builder.md

```diff
@@ -563,5 +563,5 @@ We learned the essential code for all Transform Gems using the Limit Gem as an e
 
 **How does Gembuilder work for projects aside from Python and Scala?** Package Hub supports re-usable components in SQL, for example, to create a Macro to use custom Jinja logic. A user can create a Custom Gem to be used within the same project using Macros. As of Prophecy Version 3.2, support for reusable components in SQL is in “beta” release. These are the limitations:
 
-1. A SQL project cannot be marked as package. so it wont show on package hub.
-2. User cannot reuse a model, Gem etc from one project to another for SQL projects.
+1. A SQL project cannot be marked as package. Therefore, it won't show on package hub.
+2. You cannot reuse a model or Gem from one project to another for SQL projects.
```

Comment:
You can do it. Not via the package hub. But user can release a project and add it to another projects dependencies via the dependencies setting.

<img width="1541" alt="Screenshot 2024-10-23 at 9 32 23 AM" src="https://github.com/user-attachments/assets/782d69bb-9988-4cd0-ab3f-bb8d84baef40">

Reply:
Done.